cypress/e2e/03-add_devices.cy.js

```diff
@@ -0,0 +1,110 @@
+import { generateCovidOnlyDevice, generateMultiplexDevice, loginHooks } from "../support/e2e";
```

Comment:
To address your concerns here 
> I just tested this in local (outside the docker container. yarn e2e:local) and noticed two things.
> 
> 1. This fixes a weird behavior that I had of cypress losing reference of the browser and me having to relaunch manually so that is awesome!!
> 2. I was not able to run specs 03 and 08 and those specs normally run well with the e2e:local command. Not sure if it is a coincidence or related to the changes

I added this test to create both covid and multiplex devices (before we were creating only a multiplex device).

Thank you for catching that @johanna-skylight! For me when running those tests, there was an assumption being made that I was using a covid device for those tests when I was actually using a multiplex device. Let me know if adding this test is too much especially with the device syncing work we are doing. However, I thought it would be a good solution to ensure the devices we are testing on locally and in docker are the same. Let me know what you think!! :D

Reply:
@zdeveloper @nathancrtr @DanielSass 
Do you see any issues with me adding this extra step here to create a covid only device so that when we run cypress tests locally we do not have to rely on assumptions being made about the device available for testing? 
This is similar to how we currently create a multiplex device to perform the e2e multiplex testing flow. 

Let me know if this will be an issue or unnecessary with any of the LIVD table device syncing work!